Switch to PELUX distro

GordanM opened the pull request:
The PELUX distribution is a derived poky distribution.

Signed-off-by: Gordan Markuš <gordan.markus@pelagicore.com>

Commit:
Switch to PELUX distro

The PELUX distribution is a derived poky distribution.

Signed-off-by: Gordan Markuš <gordan.markus@pelagicore.com>

zeenix reviewed (approved):
Go for it if it works!

Commit:
local.conf: remove DISTRO and IMAGE definitions

The IMAGE additions have been moved to the PELUX image bitbake recipe,
while the DISTRO additions have been moved to the PELUX distro conf file.

Signed-off-by: Gordan Markuš <gordan.markus@pelagicore.com>

Commit:
local.conf: remove licence whitelisting

Signed-off-by: Gordan Markuš <gordan.markus@pelagicore.com>
